source/mainnet/smart-contracts/tutorials/wCCD/wCCD-front-end-set-up.rst

```diff
@@ -4,6 +4,354 @@
+        to the accounts of the user and a link that points to a ``JSON-RPC server``.
+
+    The next steps are based on the pre-MVP browser wallet from a git commit before around 22.8.2022.
+    These earlier versions of the pre-MVP browser wallet have an input field for the private key
+    to import an already existing account and an input field for the link to the ``JSON-RPC server``.
+    Alternatively, you can follow workflow 2 to use the most recent pre-MVP browser wallet
+    without hosting your own node and ``JSON-RPC server``.
+
+    You are ready now to start the browser wallet by clicking on the Concordium icon at the top right of the
+    ``Chrome`` browser.
+
+    .. image:: ./images/wCCD_tutorial_12.png
+        :width: 100 %
+
+    .. note::
+
+        The puzzle icon at the top right of the ``Chrome`` browser allows you to manage your browser extensions.
+        You can enable pinning of the Concordium browser wallet.
+
+        .. image:: ./images/wCCD_tutorial_13.png
+            :width: 30 %
+
+    .. dropdown:: Getting your private key from an account already imported to the `concordium-client`
+
+        Display your keys with the following :ref:`command <concordium-client-display>`
+
+        .. code-block:: console
+
+            $./concordium-client config show
+
+        Save the ``encryptedSignKey`` blob to a file named ``output.json``. The content of that file
+        should look similar to the below content.
+
+        .. code-block:: json
+
+            {
+                "cipherText": "K1ylur5Qy+UUYlwyShu1W6rRgRhcN12e91SEGZ9UBboEzTVVQ80cDpsJNBQmU+sBlo1FKrGxKFzPjxhKxxohmZ99yDXgyo9bMDxuTosqcfY=",
+                "metadata": {
+                    "encryptionMethod": "AES-256",
+                    "initializationVector": "oJhcClLqUEotJxh4nmuCgA==",
+                    "iterations": 100000,
+                    "keyDerivationMethod": "PBKDF2WithHmacSHA256",
+                    "salt": "0XSYLtrsLN+XXwYqxD+gDw=="
+                }
+            }
+
+        Download the :ref:`utils tool <downloads-testnet-auxiliary-tools>` under the auxiliary tools section.
+        This tool is able to decode your encrypted key.
+
+        You can find additional information on the `utils` tool :ref:`here <developer-tools>`.
+
+        Decode your private key by running the decrypt command on the ``output.json`` file.
+        You will need to enter your password from the backup file when it was exported from the mobile wallet.
+
+        .. code-block:: console
+
+            $./utils decrypt --in output.json --out decrypted.example
+
+        Your private key will be saved to the ``decrypted.example`` file.
+
+    Enter the below ``JSON-RPC`` endpoint into the browser wallet to connect to
+    your local ``JSON-RPC server`` on port 9095.
+
+    .. code-block:: console
+
+        http://127.0.0.1:9095
+
+    In case you run an older pre-MVP wallet, you have to enter the private key and
```

Comment:
I think we can assume that people will use a new pre-MVP wallet. So we should remove the extra steps needed for older versions.

Reply:
remove these sentences